### ISSUE TITLE
Question on RGBD camera

### ISSUE THREAD

**Issue by RyodoTanaka** [usnistgov/ARIAC#20]
I could not re-open the following issue.
https://github.com/usnistgov/ARIAC/issues/12#issuecomment-599928372
Thus, I will copy and paste my question on here.

@Pavel-P
Hi, I have a question on depth_camera and rgbd_camera.
I understand that you update depth_camera can only publish PointCloud topics.
Before the update, we could subscribe also RGB data.
But because of RGBD camera update, there are no sensor can publish RGB data except `rgbd_camera`.
I can apply this modify and I understand your great effort for making the rule fair.
But I am wondering that this year's task, we need to detect the color.
And we have no choice to detect them except most expensive cost sensors(`rbgd_camera`).
Thus, if possible, I would like to use simple rgb camera as low cost sensor ($100 cost sensor).
If not possible, to utilize `rgbd_camera` makes no sense because, the cost is same as `logical_camera` which returns TF and color.
I think, in the following source code, simple rgb camera can use. But I could not launch them with `type: camera` parameter.
https://github.com/usnistgov/ARIAC/blob/master/nist_gear/script/gear.py#L62
In anyway, I would like to ask you about the answer for using simple rgb camera.

Thank you for your kind.
Best Regards,
Ryodo

**Comment by RyodoTanaka** [comment]
Could someone please answer this ... ?

**Comment by zeidk** [comment]
We apologize for not replying to this issue yet. We will try to provide an answer today.

**Comment by Pavel-P** [comment]
The previous version of the depth camera was implemented as a laser scanner, so I don't think it was possible to subscribe to any RGB data prior to introducing the RGBD camera. I investigated where that `type: camera` parameter came from and I couldn't find any associated sensor with it in previous competitions, so I think it is there on accident.
Besides that, I completely agree with your reasoning about needing a way to detect color with a sensor other than the logical camera (this was the original reason for adding the RGBD camera). I also agree that there is no reason to choose an RGBD camera over the logical camera at the current cost. The only reason I chose this solution was because we failed to specify ahead of time that we are introducing additional sensors this year. Because of this, we can't introduce any additional sensors (including RGB) that cost less than the logical camera without creating unfair conditions.
